code/modules/clothing/suits/jobs.dm

```diff
@@ -39,7 +39,7 @@
 	inhand_icon_state = "chef"
 	permeability_coefficient = 0.5
 	body_parts_covered = CHEST|GROIN|ARMS
-	allowed = list(/obj/item/kitchen, /obj/item/knife/kitchen)
+	allowed = list(/obj/item/kitchen, /obj/item/knife/kitchen, 	/obj/item/storage/bag/tray)
```

Comment:
ditto

Reply:
Fixed, but will post it later as I have no Ethernet currently and writing from phone